package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-    "amqp": "git://github.com/postwait/node-amqp.git#ae3a1435ac40379cb6f395085f3dc4444b649143",
+    "amqp": "https://github.com/postwait/node-amqp.git#ae3a1435ac40379cb6f395085f3dc4444b649143",
```

Comment:
Because it's a hash you might be okay, but if it fails building, using git+https:// instead will likely help.